src/chapters/installing.xml

```diff
@@ -136,10 +136,14 @@
                                         <colspec colname="c2" colnum="2" colwidth="1.0*"/>
                                         <tbody>
                                             <row>
-                                                <entry>Debian / Ubuntu package</entry>
+                                                <entry>Debian package</entry>
                                                 <entry><package>libjpeg62-turbo-dev</package></entry>
                                             </row>
                                             <row>
+                                                <entry>Ubuntu package</entry>
+                                                <entry><package>libjpeg-turbo8-dev</package></entry>
```

Comment:
Ubuntu uses Debian as the basis of each of their releases, right? Are you sure that Debian vs. Ubuntu is the dividing line between `libjpeg62-turbo-dev` and `libjpeg-turbo8-dev`?